cmd/gapis/main.go

```diff
@@ -107,8 +106,9 @@ func run(ctx context.Context) error {
 	return server.Listen(ctx, *rpc, server.Config{
 		Info: &service.ServerInfo{
 			Name:         host.Instance(ctx).Name,
-			VersionMajor: uint32(version.Major),
-			VersionMinor: uint32(version.Minor),
+			VersionMajor: uint32(app.Version.Major),
+			VersionMinor: uint32(app.Version.Minor),
+			VersionPoint: uint32(app.Version.Point),
```

Comment:
Humm, I can't think of a good reason to right now. Happy to add it if we find a usage case.